src/InvestmentManager.sol

```diff
@@ -189,7 +192,11 @@ contract InvestmentManager is Auth {
     ///         in case their orders got fulfilled.
     /// @dev    The user tranche tokens required to fulfill the redemption request have to be locked,
     ///         even though the currency payout can only happen after epoch execution.
-    function requestRedeem(address liquidityPool, uint256 trancheTokenAmount, address user) public auth {
+    function requestRedeem(address liquidityPool, uint256 trancheTokenAmount, address user)
```

Comment:
like `requestDeposit`, I think renaming `user` to `operator` is more descriptive and consistent